.github/ISSUE_TEMPLATE/1-bug.yml

```
@@ -16,82 +17,86 @@ body:
-      label: Link to runnable example
-    id: runnable-example
-    type: input
-    validations:
-      required: false
   - attributes:
       description: |
         How did this happen?
-        Please provide a [minimal and reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
+
+        Provide a [minimal and reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
+        Link to a repository or sandbox or show the code here.
+        Remove everything that is not needed.
+        Use plain Node and npm if possible.
       label: Steps to reproduce
     id: steps-to-reproduce
     type: textarea
     validations:
       required: true
   - attributes:
-      description: What should happen?
-      label: Expected behavior
-    id: expected-behavior
+      description: |
+        What happens?
+        What is wrong with the current behavior?
+      label: Actual behavior
+    id: actual-behavior
     type: textarea
     validations:
       required: true
   - attributes:
-      description: What happens instead?
-      label: Actual behavior
-    id: actual-behavior
+      description: |
+        What should happen instead?
+        Why should that happen?
+        Why is that better than the current behavior?
+      label: Expected behavior
+    id: expected-behavior
     type: textarea
     validations:
       required: true
   - attributes:
-      label: Affected runtime and version
+      label: Runtime
       description: |
-        What runtime are you using (such as `node@18.2` or `deno@1.12.2`)?
-        Please test using the latest version.
+        What runtime are you using?
+        Please test using latest versions.
+        Example: `node@23.2.0` or `deno@2.0.6`.
     id: runtime
     type: input
     validations:
-      required: true
+      required: false
   - attributes:
       description: |
-        What package manager are you using (such as `npm@8.9.0`)?
-        Please test using the latest version.
-      label: Affected package manager and version
+        What package manager are you using?
+        Please test using latest versions.
+        Example: `npm@10.9.0`.
+      label: Package manager
     id: package-manager
     type: input
     validations:
       required: false
   - attributes:
       description: |
-        What operating system are you using (such as `macOS Monterey 12.4`)?
-        Please test using the latest version.
-      label: Affected OS and version
+        What operating system are you using?
+        Please test using latest versions.
+        Example: `macOS Sequoia 15.1.1`.
+      label: Operating system
     id: operating-system
     type: input
     validations:
       required: false
   - attributes:
       description: |
         What tools are used to build or bundle the project?
-        Please test using the latest version.
+        Please test using latest versions.
```

Nitpicky, but I think this is better. Feel free to ignore.

```suggestion
        Please test using the latest versions.
```

---

/cc @ChristianMurphy as a native English speaker, strong opinions?